src/phylum/ci/cli.py

```diff
@@ -179,10 +175,10 @@ def get_args(args: Optional[Sequence[str]] = None) -> Tuple[argparse.Namespace,
             detail: https://docs.phylum.io/docs/phylum-package-score#risk-domains""",
     )
     threshold_group.add_argument(
-        "-u",
+        "-v",
```

Comment:
Even though this is a breaking change, it shouldn't matter since the change to policy based analysis will remove all these threshold options.

Reply:
```suggestion
```

There's no point in using `-v` here if you know that you want to use it for `--verbose` soon. It's simpler to force usage of the long form until the argument goes away next week.

Reply:
Good point...changed.